tests/test_uv_redirect_boundary.py

```
@@ -18,34 +18,12 @@ def clear_trusted_uv_opener_cache() -> Iterator[None]:
-    )
-
-    assert followed is not None
-    assert followed.full_url == allowed
-
-
 def test_trusted_uv_redirect_handler_rejects_before_following() -> None:
-    """Non-allowlisted hops are rejected before urllib creates a target request."""
-    handler = materializer._TrustedUvReleaseAssetRedirects()
+    """Every HTTP redirect is rejected before urllib creates a target request."""
+    handler = materializer._RejectTrustedUvRedirects()
     original = urllib.request.Request(materializer.TRUSTED_UV_ARCHIVE_URL)
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)